### ISSUE TITLE
Typescript - needless module declaration

### ISSUE THREAD

**Issue by mauriciovigolo** [vitaly-t/pg-monitor#10]
@vitaly-t, today I tried the recent version of pg-monitor, after the last commit - typescript 2 support. 

I had problems to import the library because of the module declaration. In the way it is, it's happening the following:

```Typescript
import * as pgMonitor from 'pg-monitor';

pgMonitor.pgMonitor.attach(options); .. 
```
This situation is described in [typescript - namespaces and modules doc](https://www.typescriptlang.org/docs/handbook/namespaces-and-modules.html#needless-namespacing). 

--- 
Complementing, I tried to use as:

```Typescript
import { pgMonitor } from 'pg-monitor';

pgMonitor.attach(options); .. 
```
This will result an exception during runtime - "TypeError: Cannot read property 'attach' of undefined"

I will open a PR, ok?

**Comment by mauriciovigolo** [comment]
Hi @vitaly-t, any news about this issue?

**Comment by vitaly-t** [comment]
I will make time for it soon ;) That's plenty of changes you made, well done! :wink:

**Comment by mauriciovigolo** [comment]
Great @vitaly-t! No problem. Having any problems just post your comments and I will do the corrections. By the way, I didn't have time to code some examples about that pg-promise issue, regarding custom 
promises for Typescript, but I will reserve some time to try to help in that. Thanks!

**Comment by vitaly-t** [comment]
@mauriciovigolo Thanks again! The update has been released with [v0.7.1](https://github.com/vitaly-t/pg-monitor/releases/tag/v.0.7.1).

**Comment by mauriciovigolo** [comment]
@vitaly-t glad I could help. Thanks for your outstanding libraries!!
